pkg/util/progressdots/progressdots.go

```diff
@@ -25,14 +25,14 @@ const DefaultInterval = 2 * time.Second
 
 type ProgressDots struct {
 	interval time.Duration
-	handler  chan bool
+	handler  chan struct{}
```

Comment:
Any benefit in making it struct ?

Reply:
@budhrg https://dave.cheney.net/2014/03/25/the-empty-struct

Reply:
I see no real benefit. It is not like we consume hundreds of booleans, for different channels. It is just a bool.

This obscures code...